src/main/java/net/sf/jabref/logic/formatter/casechanger/Title.java

```diff
@@ -16,6 +17,11 @@ public Title(String title) {
         this.words.addAll(new TitleParser().parse(title));
     }
 
+    public List<Word> getWordsReadOnly() {
```

Comment:
And please add at least a comment in `Word` that this class should be rewritten to be not mutable.